src/controllers/ExperimentViewController.ts

```diff
@@ -109,6 +113,14 @@ export class ExperimentViewController {
     }
   }
 
+  /**
+   * Generates a random interval time between two bounds set in state.
+   */
+  generateRandomInterval(): number {
+    const bounds = this.experiment.intervalTimeBounds
+    return (Math.floor(Math.random() * bounds.max) + bounds.min) * 1000
```

Comment:
Maybe easier to understand.

```suggestion
    return Math.min(bounds.max, Math.max(bounds.min, Math.random * 1000))
```

Reply:
Yeah good suggestion!